packages/mosaic/sql/src/init.ts

```diff
@@ -0,0 +1,5 @@
+import { setDefaultVisitor } from './ast/node.js';
+import { duckdbVisitor } from './visit/duckdb-visitor.js';
+
+// Initialize the default visitor
+setDefaultVisitor(duckdbVisitor);
```

Comment:
This causes a side effect which is probably not good. If we are okay with the general approach in this pull request, I can remove this and change the rest of mosaic to initialize the visitor.